x-pack/plugins/reporting/common/types.ts

```diff
@@ -57,22 +57,29 @@ export interface ReportOutput {
 
 export type TaskRunResult = Omit<ReportOutput, 'content'>;
 
+export interface BaseParams {
+  layout?: LayoutParams;
+  objectType: string;
+  title: string;
+  browserTimezone: string; // to format dates in the user's time zone
+  version: string; // to handle any state migrations
+}
+
+// base params decorated with encrypted headers that come into runJob functions
+export interface BasePayload extends BaseParams {
```

Comment:
Moved this interface from server so it can replace an inline definition of the same interface,  which was inside of ReportSource